src/helper/cookie/index.ts

```diff
@@ -57,3 +57,8 @@ export const setSignedCookie = async (
 export const deleteCookie = (c: Context, name: string, opt?: CookieOptions): void => {
   setCookie(c, name, '', { ...opt, maxAge: 0 })
 }
+
+export const getCookieValue = (cookie: string, key: string): string => {
```

Comment:
Since `getCookieValue()` is only used in `src/client/client.test.ts`, you can define it in `src/client/client.test.ts`. You should not include it in the helper to add unnecessary APIs or keep it in size.

Reply:
@yusukebe 
Thank you for comment.
I removed it from helper.